### DISCUSSION TITLE
Link Headers

### DISCUSSION THREAD

**Discussion by stranger-danger-zamu** [meilisearch/product#475] [Feedback & Feature Proposal]
No reason to have every client calculate the next page link based on the number of hits and the current offset. Passing the proper [Link Heaader](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Link)s should just be table stakes. It makes it trivial to see if there is more content to consume just by checking for the existence of a rel="next" Link Header.

GitHub even has a whole page on how to use them: https://docs.github.com/en/rest/guides/traversing-with-pagination

**Comment by gmourier** [comment]
Hi @stranger-danger-zamu! 👋

We evaluated this possibility at one point but were afraid that it would be complicated to use for novice users.

An idea could be to add an `isMore` in the response.

Is it just a frustration to parse the JSON body response? We could think about adding these headers also.
